package/src/components/CanvasZoom/CanvasZoom.test.jsx

```diff
@@ -1,14 +1,18 @@
 import { mockCanvasProvider } from '@/mocks/mockCanvasProvider'
 import { MockCanvas, MockPoint, mockFabric } from '@/mocks/mockFabric'
+import { mockReactRedux } from '@/mocks/mockReactRedux'
 import React from 'react'
 import { shallow } from 'enzyme'
-import { CanvasZoom as ProviderMock } from './CanvasZoom'
+import { CanvasZoom } from './CanvasZoom'
 
 jest.mock('fabric', () => mockFabric)
-
 jest.mock('@/components/CanvasProvider', () => mockCanvasProvider)
+jest.mock('react-redux', () => mockReactRedux)
```

Comment:
looks as redundant. Cause actual component is not connected to redux

Reply:
Removed